src/components/Form/CreateActivationForm/CreateActivationForm.tsx

```diff
@@ -21,7 +21,7 @@ const CreateActivationForm = () => {
   const history = useHistory();
   const { triggerTooltip } = useTooltip();
 
-  const { mutate, isLoading } =
+  const { mutate, isPending: isLoading } =
```

Comment:
Consider omitting the renaming and using `isPending` as-is for clarity

```suggestion
  const { mutate, isPending } =
```